apps/modernization-ui/src/apps/page-builder/pages/EditPage/EditPageTabs/EditPageTabs.tsx

```diff
@@ -0,0 +1,177 @@
+
+    const handleUpdateTab = async () => {
+        if (pageId) {
+            updateTab(
+                token,
+                parseInt(pageId),
+                {
+                    name: tabDetails.name,
+                    visible: tabDetails.visible
+                },
+                selectedEditTab!.id!
+            )
+                .then(() => {
+                    setEditSuccess(true);
+                    setIsEditing(false);
+                    onAddSuccess();
+                })
+                .catch((e) => {
+                    console.error(e);
+                });
+        }
+    };
+
+    const resetEditPageTabs = () => {
+        setSelectedEditTab(undefined);
+        setAddSuccess(false);
+        setEditSuccess(false);
+    };
+
+    return (
+        <>
+            <div className="edit-page-tabs" data-testid="edit-page-tabs">
+                {tabs &&
+                    tabs.map((tab, i) => {
+                        return (
+                            tab.visible && (
+                                <div
+                                    key={i}
+                                    className={`edit-page-tabs__tab ${active === i ? 'active' : ''}`}
+                                    onClick={() => setActive(i)}>
+                                    <h4>{tab.name}</h4>
+                                </div>
+                            )
+                        );
+                    })}
+                <ModalToggleButton unstyled type="button" modalRef={modalRef} data-testid="openManageTabs">
+                    <div className="edit-page-tabs__tab add">
+                        <Icon.Edit />
+                        <h4>Manage tabs</h4>
+                    </div>
+                </ModalToggleButton>
+            </div>
+            <ModalComponent
+                size={'tall'}
+                modalRef={modalRef}
+                modalHeading={
+                    !isAdding ? (
+                        <div className="manage-tabs-header">
+                            <div>Manage tabs</div>
+                            <Button className="" type="button" onClick={() => setIsAdding(true)}>
+                                <Icon.Add className="margin-right-05em add-tab-icon" />
+                                <span>Add new tab</span>
+                            </Button>
+                        </div>
+                    ) : (
+                        'Manage tabs'
+                    )
+                }
+                modalBody={
+                    <div className="edit-page-tabs__modal--body">
+                        {addSuccess ? (
+                            <AlertBanner type="success">You've successfully Added a new tab!</AlertBanner>
+                        ) : editSuccess ? (
+                            <AlertBanner type="success" onClose={() => resetEditPageTabs()}>
+                                You've successfully saved your changes to <b>&nbsp;{tabDetails?.name}!</b>
```

Comment:
Issue: this is displaying oddly
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/cf19936d-2da3-412b-a1ac-0fe36a1256e5)